components/mediation-monitor/mediation-data-publisher/org.wso2.carbon.das.messageflow.data.publisher/src/main/java/org/wso2/carbon/das/messageflow/data/publisher/publish/DataBridgePublisher.java

```diff
@@ -0,0 +1,76 @@
+package org.wso2.carbon.das.messageflow.data.publisher.publish;
```

Comment:
Shall we add the license header for this file

Reply:
opps! missed it. Will add.